controllers/disruption_controller_test.go

```diff
@@ -168,12 +168,6 @@ var _ = Describe("Disruption Controller", func() {
 				Selector:   map[string]string{"foo": "bar"},
 				Containers: []string{"ctn1"},
 				Duration:   "10m",
-				NodeFailure: &chaosv1beta1.NodeFailureSpec{
```

Comment:
Why did these need to be removed?

Reply:
this disruption is blocked by the ddmark validation -- as it should be, it's not allowed by the rules we set up
should I change the e2e test structure to try and get better e2e coverage following this update ?